src/ApplicationCore/Interfaces/IFraudPreventionService.cs

```diff
@@ -42,7 +42,7 @@ public interface IFraudProtectionService
 
         Task<Response> PostSignIn(AccountProtection.SignIn request, string correlationId, string envId);
 
-        Task<Response> PostCustomAssessment(CustomAssessment assessment, string correlationId, string envId);
+        Task<Response> PostCustomAssessment(CustomAssessment assessment, string correlationId, string envId, bool useV2);
```

Comment:
left this as a bool for now, since we're only testing v1 and v2, but can be changed to the enum if we want to test all four paths eventually.